docs/content/operations/including-extensions.md

```diff
@@ -3,31 +3,47 @@ layout: doc_page
 ---
 # Including Extensions
 
-Druid uses a module system that allows for the addition of extensions at runtime.
+Druid uses a module system that allows for the addition of extensions at runtime. To instruct Druid to load extensions, follow the steps below.
 
-## Specifying extensions
+## Download extensions
 
-Druid extensions can be specified in the `common.runtime.properties`. There are two ways of adding druid extensions currently.
+Core Druid extensions are already bundled in the Druid release tarball. You can get them by downloading the tarball at [druid.io](http://druid.io/).
+Unpack the tarball, you will see an ```extensions``` folder that contains all the core extensions, along with a ```hadoop-dependencies``` folder
+where it contains all the hadoop extensions. Each extension will have its own folder that contains extension jars. However, due to license issue,
```

Comment:
Instead of "license issue" it would be more common to say "a licensing issue" or "licensing issues", depending on how many there are.